connectors/idp-extraction/src/main/java/io/camunda/connector/idp/extraction/model/ExtractionRequest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.idp.extraction.model;
+
+import io.camunda.connector.aws.model.impl.AwsBaseRequest;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+public class ExtractionRequest extends AwsBaseRequest {
```

Comment:
To be more flexible in terms of cloud providers, I would recommend to create another class in your model like `AwsRequest` and just reference it `ExtractionRequest` so you dont have to inherit from the `AwsBaseRequest` here.

Reply:
It would also allow you to change this one to a record.

Reply:
Created a new class `BaseRequest` to extend from `AwsBaseRequest `. 
Converted `ExtractionRequest ` to a record.